clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -173,12 +173,13 @@ public class TopicConfig {
     public static final String MIN_IN_SYNC_REPLICAS_DOC = "When a producer sets acks to \"all\" (or \"-1\"), " +
         "this configuration specifies the minimum number of replicas that must acknowledge " +
         "a write for the write to be considered successful. If this minimum cannot be met, " +
```

Comment:
"this configuration specifies the minimum number of replicas that must acknowledge a write for the write to be considered successful and for it to become visible to the consumers."

Reply:
Actually, this is not quite correct since it applies independently of the acks setting. Marking this as resolved.